src/unix/bsd/netbsdlike/mod.rs

```diff
@@ -439,6 +439,52 @@ pub const MNT_NODEV: c_int = 0x00000010;
 pub const MNT_LOCAL: c_int = 0x00001000;
 pub const MNT_QUOTA: c_int = 0x00002000;
 
+// sys/ioccom.h in NetBSD and OpenBSD
+pub const IOCPARM_MASK: u32 = 0x1fff;
+pub const IOCPARM_SHIFT: u32 = 16;
+pub const IOCGROUP_SHIFT: u32 = 8;
```

Comment:
I will take a look deeper, but for now, I would like to say that `IOCPARM_SHIFT` and `IOCGROUP_SHIFT` doesn't exist on OpenBSD